### PR TITLE
Android Client progress report

### PR DESCRIPTION
**Pull request by openwarrior**

### CODE REVIEW

File: src/main/jbake/content/blog/2016/07/19/android-client-progess.adoc [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+:jbake-status: published
+:jbake-tags: blog, hawkular android client, progress, alerts, explorer
+
+Hello to all. I am quite late for this port as my last post was on march 2016 and after that a lot of things have changed. Well here is progress report of Android client.
+
+== Step wise short points
+
+Below are the steps taken till now.
+
+. Tokens are removed
```
Comment (pilhuhn) [inline src/main/jbake/content/blog/2016/07/19/android-client-progess.adoc]: Can you quickly mention why? (I know it, but readers may not)

File: src/main/jbake/content/blog/2016/07/19/android-client-progess.adoc [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+== Step wise short points
+
+Below are the steps taken till now.
+
+. Tokens are removed 
+. Filter to allow and hide acknowledged alerts.
+. Seperate activity for detailed alert.
+. Reading and posting comment on alert.
+. Update Support Libs, Aerogear libs, &  ButterKnife.
+. Inventory exporer is added.
```
Comment (pilhuhn) [inline src/main/jbake/content/blog/2016/07/19/android-client-progess.adoc]: Typo explorer

File: src/main/jbake/content/blog/2016/07/19/android-client-progess.adoc [hunk 3]
```diff
@@ -0,0 +1,57 @@
+= What is new in Android Client
+Anuj Garg
+2016-7-19
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, hawkular android client, progress, alerts, explorer
+
+Hello to all. I am quite late for this port as my last post was on march 2016 and after that a lot of things have changed. Well here is progress report of Android client.
```
Comment (pilhuhn) [inline src/main/jbake/content/blog/2016/07/19/android-client-progess.adoc]: in March

Comment (pilhuhn) [inline src/main/jbake/content/blog/2016/07/19/android-client-progess.adoc]: Remove the apology (I am quite late)

File: src/main/jbake/content/blog/2016/07/19/android-client-progess.adoc [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+== Step wise short points
+
+Below are the steps taken till now.
+
+. Tokens are removed 
+. Filter to allow and hide acknowledged alerts.
+. Seperate activity for detailed alert.
+. Reading and posting comment on alert.
+. Update Support Libs, Aerogear libs, &  ButterKnife.
```
Comment (pilhuhn) [inline src/main/jbake/content/blog/2016/07/19/android-client-progess.adoc]: Superfluous ,

File: src/main/jbake/content/blog/2016/07/19/android-client-progess.adoc [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+. Inventory exporer is added.
+. Time range selection in metrics.
+. Add and remove favourite metrics.
+. Jump to associated metric from explorer.
+
+
+== Detailed Alert Screen
+
+Detailed screen for alert is useful part get a detailed information about alert, rource associated, trigger that caused it and state of alert etc. Now you can also add comment/note to alerts.
```
Comment (pilhuhn) [inline src/main/jbake/content/blog/2016/07/19/android-client-progess.adoc]: a useful part to (get a detailed..)?

Comment (pilhuhn) [inline src/main/jbake/content/blog/2016/07/19/android-client-progess.adoc]: rource -> resource?  Perhaps even /associated resource/ ?

File: src/main/jbake/content/blog/2016/07/19/android-client-progess.adoc (hunk 4 above)
Comment (openwarrior) [inline src/main/jbake/content/blog/2016/07/19/android-client-progess.adoc]: Removed :)

File: src/main/jbake/content/blog/2016/07/19/android-client-progess.adoc (hunk 1 above)
Comment (openwarrior) [inline src/main/jbake/content/blog/2016/07/19/android-client-progess.adoc]: Now enough or I should place in separate paragraph?

**Comment by pilhuhn** [comment]
Thanks @anuj1708

**Pull request merged**